src/python/doc/nerve_gic_complex_user.rst

```diff
@@ -9,21 +9,27 @@ Definition
 
 .. include:: nerve_gic_complex_sum.inc
 
-Visualizations of the simplicial complexes can be done with either
+We provide two classes for computing cover complexes: one is CoverComplex, which can compute Mapper, Nerve and Graph Induced complexes in a scikit-learn format, 
+the other is NGIComplex, which only computes Nerve and Graph Induced complexes, but is a bit more flexible for input types (it can read inputs from paths to files for instance,
```

Comment:
I would add link to the class documentation as this:
```suggestion
We provide two classes for computing cover complexes: one is :class:`~gudhi.sklearn.CoverComplex`, which can compute Mapper, Nerve and Graph Induced complexes in a scikit-learn format, 
the other is :class:`~gudhi.NGIComplex`, which only computes Nerve and Graph Induced complexes, but is a bit more flexible for input types (it can read inputs from paths to files for instance,
```

Reply:
Thanks! Done.